app/models/heya/campaign_membership.rb

```diff
@@ -52,9 +52,9 @@ class CampaignMembership < ApplicationRecord
     scope :to_process, ->(now: Time.now, user: nil) {
       upcoming
         .where(<<~SQL, now: now.utc, user_type: user&.class&.base_class&.name, user_id: user&.id)
-          ("heya_campaign_memberships".last_sent_at <= (TIMESTAMP :now - make_interval(secs := "heya_steps".wait)))
+          ("heya_campaign_memberships".last_sent_at <= (:now::timestamp - make_interval(secs := "heya_steps".wait)))
           AND (
-            (:user_type IS NULL OR :user_id IS NULL)
+            (:user_type::text IS NULL OR :user_id::int IS NULL)
```

Comment:
Can this change be omitted? We use strings for user IDs and this cast breaks Heya for us.

Reply:
I've changed the query to remove the need for type casting. Let me know if this works for you now

Reply:
👍  It works, thanks for addressing this quickly!